Source/WebKit/UIProcess/AuxiliaryProcessProxy.cpp

```diff
@@ -327,7 +327,6 @@ void AuxiliaryProcessProxy::didFinishLaunching(ProcessLauncher* launcher, IPC::C
 
 #if PLATFORM(MAC) && USE(RUNNINGBOARD)
     m_lifetimeActivity = throttler().foregroundActivity("Lifetime Activity"_s).moveToUniquePtr();
```

Comment:
Wouldn't this cause us to immediately take a foreground assertion first?

Reply:
Yes, it will, but I think this is OK. This patch is only changing the order in which the jetsam and foreground assertion are taken. The assertions are taken async shortly after calling `m_throttler.didConnectToProcess(*this)` and `m_boostedJetsamAssertion = ProcessAssertion::create(*this, "Jetsam Boost"_s, ProcessAssertionType::BoostedJetsam)` on the main thread.

Thanks for reviewing!